### PR TITLE
add swift support

### PR DESCRIPTION
**Pull request by lazywhite**

### CODE REVIEW

File: autoload/SingleCompile/templates/swift.vim [hunk 1]
```diff
@@ -0,0 +1,27 @@
+" Copyright (C) 2010-2012 Hong Xu
```
Comment (xuhdev) [inline autoload/SingleCompile/templates/swift.vim]: You sure I am the copyright holder :)

Comment (xuhdev) [reply autoload/SingleCompile/templates/swift.vim]: Also, the year should be 2018.

**Comment by xuhdev** [comment]
Can you re-create the last commit to make a more meaningful commit message?

**Comment by lazywhite** [comment]
all fixed, sorry about the quality of PR

**Comment by xuhdev** [comment]
Thanks!

**Pull request merged**